source/release-notes/4-0-0.rst

```diff
@@ -80,17 +98,38 @@ Updates
 * RTE improvements
   Added support for pasting from MS Word to RTE
 
+* Added friendly labels for content types and mime types on the search filter |br|
+  See `here <https://github.com/craftercms/craftercms/issues/5137>`__ for more information
+
+* Fixed data source plugin failure when loading in content form |br|
+  See `here <https://github.com/craftercms/craftercms/issues/5307>`__ for more information
+
+* Improved visuals when search is running |br|
+  See `here <https://github.com/craftercms/craftercms/issues/5135>`__ for more information
+
+* The entire project is now published (bulk publish) when an item is published for the very first time |br|
+  See `here <https://github.com/craftercms/craftercms/issues/5207>`__ for more information
+
+* Fixed format for dates when copying |br|
+  See `here <https://github.com/craftercms/craftercms/issues/5268>`__ for more information
+
+* Improved handling of blank fields |br|
+  See `here <https://github.com/craftercms/craftercms/issues/5366>`__ for more information
+
```

Comment:
Add a new regarding better locale support where things like time format respect the user's locale settings.